src/shapes/SurfaceEllipse.js

```diff
@@ -85,6 +85,8 @@ define([
                 },
                 set: function(value) {
                     this.stateKeyInvalid = true;
+                    this.resetBoundaries();
+                    // this._boundaries = null;
```

Comment:
What's the motivation for leaving this in? If it's no longer necessary, remove it. Commented out code leaves room for ambiguity in future development.

Reply:
I intended to remove them, but I mistook while searching for them. Will do.